Games/types/Mafia/roles/cards/WinByStealingClovers.js

```diff
@@ -26,14 +26,12 @@ module.exports = class WinByStealingClovers extends Card {
         if (this.game.cloversSpawned) {
           return;
         }
-
-        let numCloversToSpawn = Math.round(
-          Math2.lerp(this.data.cloverTarget, this.game.players.length, 0.8)
-        );
-
         let eligiblePlayers = this.game.players.filter(
           (p) => p.role.name !== "Leprechaun"
         );
+
+        let numCloversToSpawn = this.data.cloverTarget + (this.game.players.length - eligiblePlayers.length);
```

Comment:
this is just 3 + numLeprechaun

Reply:
For now it's identical, but if more roles were filtered from eligiblePlayers in the future, then the way it's written now will make this file easier to expand with only another line of code per role.

Reply:
don't we want numCloversToSpawn to always be 3 + numLeprechaun? filtering more roles from eligiblePlayers will mess this up